### DISCUSSION TITLE
[Unstable] AI under husk symbiote effect (the one with zealot robes) refuse to attack husked creatures (AND IM NOT INSANE)

### DISCUSSION THREAD

**Discussion by PanmanS** [FakeFishGames/Barotrauma#13659] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Here's example of bot on top cannon refusing to shredder crawlers
https://youtu.be/Wu4ljlOuFYg

### Reproduction steps

1. quickstart on any sub, like `dugong`
2. spawn AI, give em zealot robes, husk em, wait till full husk effect takes place
3. put him on any periscope/gun you like
4. spawn husked crawlers on his respective gun
5. observe AI ignoring targets

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.2.13.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

We think this could be considered a feature: it makes sense that a character who's fused with the husk would not want to kill other husks. The ability to become immune to water and pressure is so powerful that we feel it's ok that it comes with downsides like this.
